common_config/src/main/resources/crash_assistant_localization/en_us.json

```
@@ -62,72 +62,72 @@
   "gui.analysis.package_finder.header": "Scans mods to find which ones contain the specified package or class.\nThis can help identify which mod provides a specific resource.",
-  "gui.analysis.package_finder.input_message": "Enter a package or class name to search for inside mods (case-insensitive).\nSupported formats:\n- package name e.g. \"org.mozilla.javascript\"\n- class name e.g. \"IMPConfig\" or \"OkHttpClient$Builder\"\n- class name with package e.g. \"org.mozilla.javascript.BoundFunction\"",
+  "gui.analysis.package_finder.input_message": "Enter a class or package to search for inside mods (case-insensitive).\nSupported formats:\n- simple class name like 'IMPConfig' or 'OkHttpClient$Builder'\n- fully qualified class name like 'org.mozilla.javascript.BoundFunction'\n- package name like 'org.mozilla.javascript' (matches everything under this package)",
   "gui.analysis.package_finder.not_found": "'$TERM$' not found in any mod.\n",
   "gui.browser_button_tooltip": "Open link in browser",
   "gui.close": "Close",
-  "gui.codex_logs_analyser": "Codex Logs Analyser ($LANG.gui.window_name$)",
-  "gui.comment_under_title": "Post the generated message from the \"$LANG.gui.upload_all_comment$\" button in $CONFIG.text.support_place$.\nAdditionally, provide some details about how the crash happened.",
-  "gui.comment_under_title_cant_resolve": "If you can't resolve the problem by reading logs, join the $CONFIG.text.support_name$.",
+  "gui.codex_logs_analyzer": "Codex Logs Analyzer ($LANG.gui.window_name$)",
+  "gui.comment_under_title": "Post the message generated by the '$LANG.gui.upload_all_comment$' button in $CONFIG.text.support_place$.\nAdditionally, provide more details about how the crash happened.",
+  "gui.comment_under_title_cant_resolve": "If you are unable to resolve the problem by reading logs, join the $CONFIG.text.support_name$.",
   "gui.comment_under_title_pls_report": "If you want to report the problem or need help, join the $CONFIG.text.support_name$.",
   "gui.comment_under_title_screenshot_notice": "Please read the text above carefully. A screenshot of this GUI tells us nothing!",
   "gui.copied": "Copied!",
   "gui.copy_link_button": "Copy link",
-  "gui.copy_split_log_dialog_text": "The size of $FILE_NAME$ $LOG_TOO_BIG_REASON$ exceeds the limits of mclo.gs.\nTo minimize the risk of losing important information due to the log being cut off at the end,\nwe have split it into a head and tail of 25k lines or 10MB each.\n\n\n<b>Link to which part of the log would you like to $ACTION$?</b>",
-  "gui.copy_split_log_dialog_title": "Log part selection",
+  "gui.copy_split_log_dialog_text": "The size of $FILE_NAME$ $LOG_TOO_BIG_REASON$ exceeds <code>mclo.gs</code> limits.\nTo minimize the risk of losing important information when omitting the end of the log,\nwe split it into two parts of 25k lines or 10MB each: the head and the tail.\n\n\n<strong>Which part of the log would you like to $ACTION$?</strong>",
```

```suggestion
  "gui.copy_split_log_dialog_text": "The size of $FILE_NAME$ $LOG_TOO_BIG_REASON$ exceeds <code>mclo.gs</code>' limits.\nTo minimize the risk of losing important information when omitting the end of the log,\nwe split it into two parts of 25k lines or 10MB each: the head and the tail.\n\n\n<strong>Which part of the log would you like to $ACTION$?</strong>",
```

This is not a typo, it's the Saxon Genitive